server/routes/api/v1/adoptionFormRouter.js

```diff
@@ -0,0 +1,23 @@
+import express from "express"
+
+import AdoptionApplication from "../../../models/AdoptionApplication.js"
+
+const adoptionFormRouter = new express.Router()

```

Comment:
Something to consider - probably would want to call this router `adoptionApplicationRouter` naming after the model/ database table it interacts with. We use routers for all CRUD actions for a resource, so it might not always be information coming from a form - maybe we want to see all adoption applications that have been submitted! In that case it's a GET request for all adoption applications, not all _adoption forms_